parameterspace/parameterspace.py

```diff
@@ -255,6 +255,14 @@ def get_parameter_names(self) -> List[str]:
         """
         return list(self._parameters.keys())
 
+    def get_constant_names(self) -> List[str]:
+        """Get names of all constants already in the current `ParameterSpace`.
+
+        Returns:
+            Constant names.
```

Comment:
It feels like the doc string adds little information to the method name `get_constant_names()`, while I myself still needed to look up, what "constants" actually are in context of ParameterSpace :see_no_evil: .

What about something along the following lines?:
```
"""Get names of the parameters set to a constant value via the `fix()` method.

Returns:
    Names of constants in the current `ParameterSpace`.
"""
```

Reply:
Thanks @dynobo for the feedback. The more explicit docstring is indeed better, done.